ios/Podfile.lock

```diff
@@ -4,18 +4,18 @@ PODS:
     - CustomerIOMessagingInApp (~> 2.1.0-beta.2)
     - CustomerIOTracking (~> 2.1.0-beta.2)
     - React-Core
-  - CustomerIO/MessagingPushAPN (2.1.0):
-    - CustomerIOMessagingPushAPN (= 2.1.0)
-  - CustomerIOCommon (2.1.0)
-  - CustomerIOMessagingInApp (2.1.0):
-    - CustomerIOTracking (= 2.1.0)
+  - CustomerIO/MessagingPushAPN (2.1.1):
```

Comment:
App's cocoapods updated to iOS SDK 2.1.1 which contains the compile iOS bug fix.